components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -16,6 +16,30 @@ import {
 import { useAuth } from "../../../lib/authContext";
 import ExpandableContainer from "../ExpandableContainer";
 
+export const returnWrapStyling = (
```

Comment:
something to think about here - do other files in the app need to use these functions? if so we should "export" them so they are available all over our app.. if not we can just keep them local (i.e remove the export)

in this case, the function takes in an assignment-specific data type and returns styling specific to this component, so I think it makes sense to just contain it to this file

if we did plan on making this function more generic, i.e use it to style any div, and if we have several other functions like this, I'd say we should create a separate helper functions file and export it from that file

Reply:
you're right! removed the export because the files need only to be used locally.